### PR TITLE
Add PR and issues templates

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Add PR and issues templates.
It's pretty the same templates https://github.com/devfile/kubernetes-api has.

### What issues does this PR fix or reference?
N/A

### Is your PR tested? Consider putting some instruction how to test your changes
No testing it needed. It's just GitHub repo settings.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
Name: Bug report
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+about: Create a report to help us improve
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
About: Create a report to help us improve
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+title: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]:
```suggestion
Title: ''
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+labels: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```Suggestion
labels: ''
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+assignees: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]:
```suggestion
Assignees: ''
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]:
```suggestion
Name: Feature request
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+about: Suggest an idea for this project
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]:
```suggestion
About: Suggest an idea for this project
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]:
```suggestion
Title: ''
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+labels: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
Labels: ''
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+assignees: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]:
```suggestion
Assignees: ''
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
+---
+name: Task 🔧
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/task.md:2]:
```suggestion
Name: Task 🔧
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+about: Internal things, technical debt, and to-do tasks to be performed.
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/task.md:3]:
```suggestion
About: Internal things, technical debt, and to-do tasks to be performed.
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+title: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/task.md:4]:
```suggestion
Title: ''
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+labels: 'kind/task'
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/task.md:5]:
```suggestion
Labels: 'kind/task'
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+assignees: ''
```
Comment (wtam2018) [inline .github/ISSUE_TEMPLATE/task.md:6]:
```suggestion
Assignees: ''
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (sleshchenko) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: I assume it should work but I'm not sure. lowercased name is referenced in github doc 
> To be included in the community profile checklist, issue templates must be located in the .github/ISSUE_TEMPLATE folder and contain valid name: and about: YAML front matter.
> from https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

@wtam2018 Could you elaborate more why do you propose that changes?
P.S. Seems odo repo uses `name` and `about` as well https://raw.githubusercontent.com/openshift/odo/master/.github/ISSUE_TEMPLATE/Documentation.md

Comment (wtam2018) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: @sleshchenko  Nevermind those suggestions.   Those tags in fact need to be all lowercases.

**Pull request merged**